### DISCUSSION TITLE
How can i prevent renovate from creating prs for certain versions?

### DISCUSSION THREAD

**Discussion by dabaus** [renovatebot/renovate#25572] [Request Help]
### How are you running Renovate?

In bitbucket.

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

For example, if i have the version 1.9.0-rc1 then PRs should not automatically be created for all apps.
However, if renovate see 1.9.0 without the rc1, then behaviour should be as it is today. 

The problem we have is that sometimes bugs slip out, and is merged into all apps in the test environment,
and later we have to spend a lot of time to roll everything back. 

### What you tried so far.

I looked in the docs but i did not find anything relevant. How is this done?

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by dabaus** [comment]
ChatGPT gave me this. Does it work?


```
{
  "extends": ["config:base"],
  "packageRules": [
    {
      "packageNames": ["your-package-name"],
      "rangeStrategy": "widen",
      "exclude": ["*-rc*"]
    }
  ]
}
```

**Comment by rarkins** [comment]
No that won't help. It sounds like you need a package rule with an allowedVersions pattern
